src/Middleware/HttpLogging/test/FileLoggerProcessorTests.cs

```diff
@@ -46,13 +44,13 @@ await using (var logger = new FileLoggerProcessor(new OptionsWrapperMonitor<W3CL
             {
                 logger.SystemDateTime = mockSystemDateTime;
                 logger.EnqueueMessage(_messageOne);
-                fileName = Path.Combine(path, FormattableString.Invariant($"{options.FileName}{_today.Year:0000}{_today.Month:00}{_today.Day:00}.0000.txt"));
+                filePath = GetLogFilePath(path, options.FileName, _today, 0);
```

Comment:
There were a number of variables in these tests which were called "fileName-" but were referring to file paths instead. I fixed these while I was at it, but I can revert these changes if this causes too much noise in the PR.